torchtitan/models/llama3/train_configs/llama3_70b.toml

```diff
@@ -67,4 +67,6 @@ filter_fqns = ["output"]
 enabled = false
 dataset = "c4_validation"
 freq = 500
-steps = -1
+# Set steps to -1 to consume all the data in the dataset
+# WARNING: When setting to -1 there could be hangs due to mismatch among ranks
```

Comment:
Instead of adding the comment to every toml, can't we add the comment to `job_config.py`? And since you also add a runtime warning, I think it should be enough.